src/chunk.c

```diff
@@ -1720,36 +1805,50 @@ ts_chunk_exists_relid(Oid relid)
 	return ts_chunk_get_by_relid(relid, 0, false) != NULL;
 }
 
-static ScanTupleResult
-chunk_tuple_delete(TupleInfo *ti, void *data)
+/* Delete the chunk tuple.
+ *
+ * mark_as_dropped_only - instead of deleting the row, mark it as dropped.
```

Comment:
mark_as_dropped_only --> is this the old parmeter name for preserve_catalog_row

Reply:
yes